simphony_ui/ui.py

```diff
@@ -27,6 +29,34 @@ def dataset2cudssource(dataset):
     return CUDSSource(cuds=dataset)
 
 
+class CustomErrorMessage(HasStrictTraits):
+
+    # The message to be shown:
+    message = Str('Oups ! Something went wrong...')
+
+    # The time (in seconds) to show the message:
+    time = Float(20)
+
+    def show(self):
+        """ Display the error message for a limited duration.
+        """
+        view = View(
+            HGroup(
+                spring,
+                Item('message',
+                     show_label=False,
+                     style='readonly'
+                     ),
+                spring
+            ),
+            buttons=['OK'],
+            title='Error'
+        )
+
+        do_after(int(1000.0 * self.time),
+                 self.edit_traits(view=view).dispose)
```

Comment:
Well, I first used a traitsui message class: https://github.com/enthought/traitsui/blob/master/traitsui/message.py. But I stopped using it because of the tests, when running the tests with travis ci, there's no one to close the dialog and let the tests running. So I wanted a timeout.

But thinking about it, I could still use a message, and create a mock in the tests